### ISSUE TITLE
Authorization Issue

### ISSUE THREAD

**Issue by afoysal** [ratiw/vuetable-2#342]
My vuetable 2 code is like below
```
<vuetable
      api-url="http://127.0.0.1:8000/api/addresses"
      :fields="columns"
      data-path="data"
      pagination-path=""
    ></vuetable>
```

I am getting error is like below

![vueerror2](https://user-images.githubusercontent.com/1283998/34039832-5be8cf10-e1bb-11e7-87b8-ecb245c58ccf.png)

**Comment by ratiw** [comment]
This is not Vuetable error. Check your backend code. It seems like your api needs authentication.

**Comment by afoysal** [comment]
Yes, API needs authentication. I have to send authentication token from Vuetable along with `api-url`. How can I do that ?

**Comment by ratiw** [comment]
Try searching through the issues, many people have already asked and suggested solutions. Basically, you should use either `http-options` or `append-params` to attach the token to the request.
